mesh/appender.go

```diff
@@ -56,3 +58,8 @@ type Appender interface {
 	// Name returns a unique appender name and which is the name used to identify the appender (e.g in 'appenders' query param)
 	Name() string
 }
+
+// IstioStatusGetter gets the status of the istio graph components
```

Comment:
This `IstioStatusGetter` name is a little confusing to me.  It's not getting the status of just Istio, right, it's getting status for the various mesh infra components.  Would `MeshStatusGetter` or `InfraStatusGetter` make sense?

Reply:
Yes, I think `MeshStatusGetter` is more appropiate in this case.